src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -70,3 +72,15 @@ def load_command_table(self, _):
         g.custom_command('update', 'aks_agentpool_update', supports_no_wait=True)
         g.custom_command('delete', 'aks_agentpool_delete', supports_no_wait=True)
         g.custom_command('get-upgrades', 'aks_agentpool_get_upgrade_profile')
+
+    # AKS pod identity commands
+    with self.command_group('aks pod-identity', managed_clusters_sdk, client_factory=cf_managed_clusters) as g:
+        g.custom_command('add', 'aks_pod_identity_add')
+        g.custom_command('delete', 'aks_pod_identity_delete')
+        g.custom_command('list', 'aks_pod_identity_list',
+                         table_transformer=aks_pod_identities_table_format)
+        g.custom_command('add-exception', 'aks_pod_identity_exception_add')
```

Comment:
We suggest to add a command group for exception here. The commands should be like:
1. `exception add`
2. `exception delete`
3. `exception update`
4. `exception list`

And maybe an additional `exception show` is also needed ?

Reply:
we don't have `exception show`. I could create a new sub group `pod-identity exception add/delete/update/list` cc @miwithro

Reply:
```
az aks pod-identity exception -h

Group
    az aks pod-identity exception : Commands to manage pod identity exceptions in managed Kubernetes
    cluster.

Commands:
    add    : Add a pod identity exception to a managed Kubernetes cluster.
    delete : Remove a pod identity exception from a managed Kubernetes cluster.
    list   : List pod identity exceptions in a managed Kubernetes cluster.
    update : Update a pod identity exception in a managed Kubernetes cluster
```

Updated, PTAL @arrownj